### ISSUE TITLE
[Bug] "SERVER NAME" button disappears after game exit

### ISSUE THREAD

**Issue by Haisom** [redphx/better-xcloud#112]
**Platform**

 - Device: Desktop
 - OS: Windows 11
 - Browser: Edge
 - Browser Version: 115.0.1901.203 (Official build) (64-bit)
 - Better xCloud Version: 1.13.1

**Describe the bug**

Exitting a game either on Xbox ingame menu or xCloud floating menu, makes the "SERVER NAME" button next to avatar to disappear. Refreshing the tab makes it appear again.

**Screenshots/Videos**

After exit

<img width="960" alt="image" src="https://github.com/redphx/better-xcloud/assets/29853423/154d0132-cce1-4daa-bea0-5e978aaeef32">

After refresh

<img width="960" alt="image" src="https://github.com/redphx/better-xcloud/assets/29853423/46ef2bcc-122d-4a58-8e60-489ee4ffb556">

After exit

<img width="960" alt="image" src="https://github.com/redphx/better-xcloud/assets/29853423/a3011961-a309-4d2d-80b0-9b3ee7b42e50">

**Comment by redphx** [comment]
This is a known problem but I was too lazy to fill an issue. Thanks for reporting.

**Comment by keanew** [comment]
It's not a beautiful fix but you can add this line in the reset method of LoadingScreen : 
`        setTimeout(checkHeader, 2000);
`
Like this : 
```
    static reset() {
        LoadingScreen.#$waitTimeBox && LoadingScreen.#$waitTimeBox.classList.add('better-xcloud-gone');
        LoadingScreen.#$bgStyle && (LoadingScreen.#$bgStyle.textContent = '');

        LoadingScreen.#waitTimeInterval && clearInterval(LoadingScreen.#waitTimeInterval);
        LoadingScreen.#waitTimeInterval = null;
        
        setTimeout(checkHeader, 2000);
    }
```

A wait time of 2 second may not be perfect and in some case it could take more time but it works in most case :)

**Comment by redphx** [comment]
Fixed in 1.14
